src/Core/src/Handlers/SearchBar/SearchBarHandler.Android.cs

```diff
@@ -1,19 +1,32 @@
-﻿using AndroidX.AppCompat.Widget;
+﻿using System.Linq;
+using Android.Widget;
+using SearchView = AndroidX.AppCompat.Widget.SearchView;
 
 namespace Microsoft.Maui.Handlers
 {
 	public partial class SearchBarHandler : AbstractViewHandler<ISearchBar, SearchView>
 	{
+		EditText? _editText;
```

Comment:
@jsuarezruiz  I had to make this non-static as this would fail the tests because each test creates a new handler, but then this would only be set once the very first time. All updates in subsequent tests resulted in updates to this object, and not the actual nested views.